lib/blockchain_api/purger.ex

```diff
@@ -0,0 +1,15 @@
+defmodule BlockchainAPI.Purger do
```

Comment:
What do you thing about renaming this module to `BlockchainAPI.Fastly` or `BlockchainAPI.FastlyService` and have the functions be just `purge` and `post`?

Reply:
I _think_ we'd want to not specify the underlying service in the module name. In theory we could switch to a different cache service and the code calling into this module wouldn't need to change. But if we have some header utils, we could broaden this to something like `CacheService` or something like that.